### ISSUE TITLE
epub DRM removal fails (version 6.4.3)

### ISSUE THREAD

**Issue by n-simone** [apprenticeharper/DeDRM_tools#99]
I'm running Calibre v1.25 in Ubuntu 14.04.

I've had some issues throughout this process, so I'm not entirely sure where the issue is. I downloaded an .ascm file for a book I purchased. I tried to open it in Adobe Digital Editions v1.7.2 running in Wine v1.7, but got an error saying "IO Error on Network Request". The error provided a URL for the Network Path, and when I opened that in my browser, an epub file downloaded. However, when I add that file to Calibre, it retains its DRM. I tried again after running `calibre-debug -g`, and it returned the following error:

```
DeDRM v6.4.3: Trying to decrypt [book title].epub
DeDRM v6.4.3: Verifying zip archive integrity
DeDRM v6.4.3: ���[book title].epub��� is neither an Adobe Adept nor a Barnes & Noble encrypted ePub
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "/usr/lib/calibre/calibre/customize/ui.py", line 154, in _run_filetype_plugins
    nfp = plugin.run(path_to_file)
  File "calibre_plugins.dedrm.__init__", line 615, in run
  File "calibre_plugins.dedrm.__init__", line 391, in ePubDecrypt
DeDRMError: DeDRM v6.4.3: Couldn't decrypt after 0.1 seconds. DRM free perhaps?
```

**Comment by n-simone** [comment]
Update: Never mind. I was able to properly download the epub file after upgrading to ADE v4.5.1 as per the instructions here: [https://appdb.winehq.org/objectManager.php?sClass=version&iId=30928](https://appdb.winehq.org/objectManager.php?sClass=version&iId=30928)

When I tried to remove the DRM with Calibre, it still didn't work, but I soon realized that my issue was that I didn't read the instructions in the ReadMe file. After installing everything else that needed to be installed, the DRM removal worked perfectly.
